support/tests.py

```diff
@@ -25,7 +25,7 @@
 
 
 class SupportRequestsTest(TestCase):
-    fixtures = ['initial_data.json', 'moderation_test_users.json']
+    fixtures = ['moderation_test_users.json']
```

Comment:
extensions for fixtures are optional. since no other test has them, I'd recommend that we remove it here